src/nyc_trees/apps/quiz/models.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+class Quiz(object):
```

Comment:
The Django model closest related to this is `TrainingResult` in `apps.users.models`. I don't see much value in moving these but I'm open to suggestions.